.azure-pipelines/apiview.yml

```diff
@@ -7,6 +7,7 @@ trigger:
       - hotfix/*
```

Comment:
At some point we should restructure the apiview project under "tools/apiview" and move this yml file under it and call it ci.yml. We are trying to restructure this repo to make things more consistent.